build/build_suite_test.go

```diff
@@ -179,4 +181,170 @@ var _ = Describe("Test Build utilities:", func() {
 		})
 	})
 
+	Describe("Test NewEndpointSecretFromEnv", func() {
+		name := "name"
```

Comment:
missing "Context" and "It" intentionally?
i think its good to have from readability perspective.

Reply:
Updated